docs/get-details/dapps/avm/teal/index.md

```diff
@@ -12,8 +12,8 @@ Some of the opcodes in TEAL are only valid for a specific contract type. These a
 !!! warning
     When writing smart contracts, make sure to follow [TEAL guidelines](guidelines). This is very important in order to prevent smart contracts from being compromised.
```

Comment:
@barnjamin I found several additional references to consider.  Can you review https://github.com/algorand/docs/pull/878 to see if you agree with pulling it in?